spec/unit/equalizer/universal_spec.rb

```diff
@@ -146,6 +146,22 @@ def initialize(firstname, lastname)
         expect(instance.hash)
           .to eql([firstname, lastname, klass].hash)
       end
+
+      context 'when immutable' do
```

Comment:
You should keep the tests consistent, so probably a new context at the same level as the `'with keys'` context would be more appropriate. Make sure to keep the context setup consistent, so that you are testing only the public API.